docs/Migration-Limitations.md

```diff
@@ -0,0 +1,174 @@
+- [**Configuration.getApplication**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getApplication) - Concept has been discarded
+- [**Configuration.getAutoAriaBodyRole**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getAutoAriaBodyRole) - No replacement
+- [**Configuration.getCompatibilityVersion**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getCompatibilityVersion) - Concept has been discarded
+- [**Configuration.getDebug**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getDebug) - No replacement
+- [**Configuration.getFileShareSupport**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getFileShareSupport) - No replacement
+- [**Configuration.getFiori2Adaptation**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getFiori2Adaptation) - No replacement
+- [**Configuration.getFlexibilityServices**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getFlexibilityServices) - Concept has been discarded
+- [**Configuration.getFormatSettings**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getFormatSettings) - Complex replacement
+- [**Configuration.getInspect**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getInspect) - Concept has been discarded
+- [**Configuration.getManifestFirst**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getManifestFirst) - No replacement
+- [**Configuration.getNoDuplicateIds**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getNoDuplicateIds) - Concept has been discarded
+- [**Configuration.getOriginInfo**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getOriginInfo) - Concept has been discarded
+- [**Configuration.getRootComponent**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getRootComponent) - Concept has been discarded
+- [**Configuration.getStatisticsEnabled**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getStatisticsEnabled) - No replacement
+- [**Configuration.getVersion**](https://ui5.sap.com/1.136/#/api/sap.ui.core.Configuration%23methods/sap.ui.core.Configuration.getVersion) - Sync to async replacement
+
+## Sync to Async API Changes
+
+The UI5 linter cannot automatically migrate APIs that change from synchronous to asynchronous. This is a general limitation as it would require restructuring the code flow. Examples include:
```

Comment:
```suggestion
Currently, UI5 linter cannot automatically migrate APIs that execute synchronously if their replacement would execute asynchronously (i.e. returns a promise). This is a general limitation as it would require restructuring the code flow. Examples include:
```